src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -391,82 +325,84 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
-        hideLeft={displayNewBorders}
-        hideTop={displayNewBorders}
-      />
-      <Flex
+      <OakFlex
         $position="relative"
+        $borderRadius="border-radius-s"
         $alignItems={"center"}
         $justifyContent={"space-between"}
-        $gap={0}
+        $gap="space-between-none"
         $flexDirection={["column", "row"]}
         $width={"100%"}
+        $background={showSubjects || showPhases ? "grey30" : "white"}
       >
-        <Flex
+        <OakFlex
           $flexDirection={"row"}
           $alignItems={"center"}
           $justifyContent={"flex-start"}
           $width={["100%", "100%", "100%"]}
         >
-          <Box
+          <OakFlex
             $position={"relative"}
-            $width={["50%", "50%"]}
-            $borderColor={showSubjects ? "lemon" : "transparent"}
-            $ba={3}
-            $background={subjectBackground}
+            $alignSelf={"stretch"}
+            $background={showSubjects ? "white" : null}
+            style={{ width: "50%" }}
           >
-            <BoxBorders
-              $color={showSubjects ? "black" : "transparent"}
-              gapPosition="rightTop"
-            />
-
-            <SelectButton
-              $ph={24}
-              $pv={24}
-              onClick={toggleShowSubjects}
-              title="Subject"
+            <FocusIndicatorAlt
+              disableMouseHover={true}
+              subFocus={showSubjects}
+              disableActive={true}
+              $width={"100%"}
+              $bblr={["border-radius-square", "border-radius-m"]}
+              $bbrr={["border-radius-square", "border-radius-square"]}
+              $btlr={["border-radius-m", "border-radius-m"]}
+              $btrr={["border-radius-square", "border-radius-square"]}
             >
-              <OakHeading
-                tag={"h3"}
-                $font={"heading-light-7"}
-                $mb="space-between-sssx"
-                $color={!showSubjectError ? "black" : "red"}
-                data-testid="selectSubjectHeading"
-              >
-                Subject
-              </OakHeading>
-              <OakP
-                $font={"body-2"}
-                $color={!showSubjectError ? "black" : "red"}
-              >
-                {showSubjectError && (
-                  <>
-                    <Icon
-                      $color={"red"}
-                      name="content-guidance"
-                      verticalAlign="bottom"
-                    />
-                    <OakSpan>Select a subject</OakSpan>
-                  </>
-                )}
-                {selectedSubject && <>{selectedSubject.title}</>}
-                {!showSubjectError && !selectedSubject && "Select"}
-              </OakP>
-              <ButtonFocusUnderline $color={"black"} name="underline-1" />
-            </SelectButton>
-          </Box>
+              <PhaseButton onClick={toggleShowSubjects} title="Subject">
```

Comment:
Should this be named `ModalButton` or something of the like? Just a note as I think I missed this earlier too